sentry-core/src/main/java/io/sentry/core/SystemOutLogger.java

```diff
@@ -42,6 +48,18 @@ public void log(SentryLevel level, String message, Throwable throwable) {
   @Override
   public void log(SentryLevel level, Throwable throwable, String message, Object... args) {
     System.out.println(
-        String.format("%s: %s \n %s", level, String.format(message, args), throwable.toString()));
+        String.format(
```

Comment:
Makes me realize the previous code was a bit naive but don't we need to check if we have an instance in the first place?
I'm afraid we might throw if any arg is null here

Reply:
True! Good catch. Now I see that `throwable` is marked as `@Nullable` in `DiagnosticLogger#log`. I'll take care of it.

Reply:
good catch, we've never used this class, that's why not so feature-complete (missing stack traces, etc...), we're always in favor of AndroidLogger for Android.